module/ldbc-dsl/src/main/scala/ldbc/dsl/DBIO.scala

```diff
@@ -111,7 +111,7 @@ object DBIO:
       override def rollback(connection:    Connection[F])(using LogHandler[F]):             F[A] = ev.raiseError(e)
       override def transaction(connection: Connection[F])(using LogHandler[F]):             F[A] = ev.raiseError(e)
 
-  given [F[_]: Temporal]: MonadError[[T] =>> DBIO[F, T], Throwable] with
+  given [F[_]: MonadCancelThrow]: MonadThrow[[T] =>> DBIO[F, T]] with
```

Comment:
NOTE: If you implement cancellation on DBIO, I guess this could be `given [F[_]: MonadCancelThrow]:MonadCancelThrow[DBIO[F, _]]`